xeno/hive_mender_strain.md

```diff
@@ -0,0 +1,25 @@
+## Abstract
+
+The Mender Hivelord is a support role that branches off the Hivelord caste and helps support a push or hold either by spewing out clouds of smoke that either heals or removes debuffs while giving a small overshield, depending on which of the two is chosen at the time. This would provide xeno players who prefer playing support style roles more options, specifically options that do not require the two player xenos to be within touching distance. This differs from the healer drone as it's heals are tied to the smoke and are derived from it's plama stores rather than it's health.
+
+
+## Goals
+
+1. Create a strain that caters to those who enjoy playing support/healer style roles prefer to take a more active role in the frontline and providing clutch rescues if and when neede.
```

Comment:
I recommend you read up on #2 and think about how this might change Mender's implementation for heals etc. 

If you don't think any change is necessary to the design doc resolve my comment and i'll take another look at the proposal.